v3/plugins/pizzafactory/vscode-libra-move/0.0.10/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: pizzafactory
+name: vscode-libra-move
+version: 0.0.10
+type: VS Code extension
+displayName: Libra's Move Language
+title: Libra's Move Language
+description: VS Code extension that provides Libra's Move Language Server. Code is provided by SDKBox. Packaging by PizzaFactory.
+icon: https://commons.wikimedia.org/wiki/File:Libra_logo_icon.svg
+firstPublicationDate: "2019-09-24"
+category: Language
+repository: https://github.com/pizzafactory-contorno/vscode-libra-move/
+spec:
+  containers:
+    - image: "eclipse/che-remote-plugin-runner-java8:7.0.0-next"
```

Comment:
If there aren't any issues with the newer version, I'd suggest using `docker.io/eclipse/che-remote-plugin-runner-java8:next` here.

Reply:
I see. I fixed here (and same issues)  and re-pushed it.